E2a-TRAS/GL01/README.md

```diff
@@ -0,0 +1,137 @@
+# Test instructions
+
+| Documentation Code  | Test Title                           | Exchange Code | Test Code | Author          | Data Owner | Version | Date       |
+|---------------------|--------------------------------------|---------------|-----------|-----------------|------------|---------|------------|
+| IFC4.3AbRV_E2a_GL01 | (RFI) Global Positioning RFI dataset | E2a (TRAS)    | GL01      | Evandro Alfieri | RFI        | 1.0     | DD.MM.YYYY |
+
+
+## Summary (Intent)
+
+This test case addresses the **export** of the required IFC entities for the exchange of **global positioning** information.
+
+- Refer to [Test Case Imports](#Test-Case-Imports) to know the prerequisites for the present test.
+
+- The [Expected Results](#Expected-Results) section lists the material that will be used to assess the fulfilment of capabilities.
+
+- :zap: This is a test-driven process: refer to the [Validation Criteria](#Validation-Criteria) to understand what is required by the test :zap:
+
+
+
+
+## Itemised Roots
+
+The Test instruction addresses the import and export of the following IFC Entities & Concept Templates:
+
+<details><summary>IFC Entities</summary>
+
+These entities represent a test-specific subset of the wider AbRV_E2a exchange and the overall AbRV MVD. **The scope of the test shall not be used as a definitive scope of the exchange, or of the MVD**
+
+- Model setup:
+   1. IfcSite
+   1. IfcMapConversion
+   1. IfcProjectedCRS
+</details>
+
+<details><summary>Concept Templates</summary> 
+
+These concept templates represent a test-specific subset of the wider AbRV_Ex exchange and the overall AbRV MVD, that must be correctly exported to meet the validation criteria. **The scope of the test shall not be used as a definitive scope of the exchange, or of the MVD**
+
+- Project Context
+   - Project Global Positioning
+</details>
+
+
+
+
+## Model Dataset
+
+This test case utilises the dataset collected in the Dataset folder and summarised in the table below. **For more details on each item see [Dataset description](Dataset/README.md).**
+
+| Filename                | Type (format)  | Description                            |
+|-------------------------|----------------|----------------------------------------|
+| GeographicData          | figure (jpg)   | Geographic data relevant for the test  |
```

Comment:
I'd have preferred a table with text (to enable copy-paste) here.

Reply:
Added